frontend/app/model/RichEvent.scala

```diff
@@ -97,14 +96,21 @@ object RichEvent {
   }
 
   abstract class LiveEvent(image: Option[EventImage], contentOpt: Option[Content]) extends RichEvent {
-    val imgUrl = image.flatMap(_.assets.headOption).fold(fallbackImage) { asset =>
-      val file = asset.secureUrl.getOrElse(asset.file)
-      val regex = "\\d+.jpg".r
-      regex.replaceFirstIn(file, "{width}.jpg")
-    }
 
     private val widths = image.fold(List.empty[Int])(_.assets.map(_.dimensions.width))
 
+    val imageMetadata = image.map(_.metadata)
+
+    val imgUrl = image.flatMap(_.assets.headOption).fold(fallbackImage){ asset =>
+      asset.secureUrl.getOrElse(asset.file)
+    }
+
+    val availableWidths = widths.mkString(",")
+
+    val srcsetOpt = image.map(_.assets.map { asset =>
```

Comment:
You could add a test for this (to ensure it doesn't get changed later without reason).